bittensor/_tokenizer/__init__.py

```diff
@@ -18,7 +18,9 @@
 # DEALINGS IN THE SOFTWARE.
 
 from transformers import AutoTokenizer
-import bittensor 
+import bittensor
+from bittensor.utils.tokenizer_utils import prep_tokenizer
```

Comment:
Would we expect users to use the prep_tokenizer function a lot? If so, it might be good to have it as a class function as well

Reply:
Not really, it's just to prepare the server tokenizer (and the bittensor.tokenizer gets prepared automatically). So clients wouldn't need to run it, for instance.